src/test/java/com/squareup/kotlinpoet/TypeSpecTest.kt

```diff
@@ -2917,6 +2908,48 @@ class TypeSpecTest {
           |""".trimMargin())
   }
 
+  @Test
+  fun externalClassFunctionHasNoBody() {
+    val typeSpec = TypeSpec.externalClassBuilder("Foo")
+            .addFunction(FunSpec.builder("bar").addModifiers(KModifier.EXTERNAL).build())
+            .build()
+
+    assertThat(toString(typeSpec)).isEqualTo("""
+      |package com.squareup.tacos
+      |
+      |external class Foo {
+      |    fun bar()
+      |}
+      |""".trimMargin())
+  }
+
+  @Test
+  fun externalInterfaceFunctionHasNoBody() {
```

Comment:
The name of the test doesn't correspond to the contents. Would actually be great to have properties and/or functions inside to verify that they're printed properly.